lib/Web2Print/Processor/HeadlessChrome.php

```diff
@@ -27,10 +27,22 @@
 use Spiritix\Html2Pdf\Output\FileOutput;
 use Spiritix\Html2Pdf\Output\StringOutput;
 
+/**
+ * @deprecated will be removed in Pimcore 11 and replaced by Processor\Chromium
+ */
 class HeadlessChrome extends Processor
 {
     private $nodePath = '';
 
+    public function __construct(){
+        trigger_deprecation(
```

Comment:
@kingjia90 I wouldn't issue the deprecation message here, for the reasons you mentioned. I thin kit's enough if we document the fact that's going away in v11 (which is already done).

Reply:
Thank you for your suggestion, looks much better now